engine/src/vault.rs

```diff
@@ -24,6 +24,6 @@ pub mod view;
 pub use crate::vault::{
     base64::{Base64Decodable, Base64Encodable},
     crypto_box::{BoxProvider, Decrypt, DecryptError, Encrypt, Key, NCDecrypt, NCEncrypt, NCKey},
-    types::utils::{ChainId, ClientId, Id, InvalidLength, RecordHint, RecordId, VaultId},
```

Comment:
I wonder, if `ChainId` can be renamed to `RecordId` as it is used as such.

Reply:
I was wondering about that as well. I think renaming `ChainId` to `RecordId` would break the old snapshot format, but there is probably a way to solve that.

Reply:
Unless something else has changed, the `ChainId` and the `RecordId` should be the same thing; even just rewrapping the `ChainId` to be a `RecordId` would be possible.  The concept of the `ChainId` doesn't make too much sense anymore since the Records aren't Chains of versions anymore.